src/IepDetails/CurrentIepLevel.js

```diff
@@ -26,13 +26,13 @@ const CurrentIepLevel = ({ level, days, nextReviewDate }) => (
 
 CurrentIepLevel.propTypes = {
   level: PropTypes.string,
-  days: PropTypes.string,
+  days: PropTypes.number,
   nextReviewDate: PropTypes.string,
 }
```

Comment:
Oops, approved this before eagle-eyed @andrewrlee spotted that actually, the value here is meant to be a string - this is the '1 year, 2 days' value.

Reply:
Andrew has just told me he's fixed the type of value that gets passed, locally, so I've reverted that part of the change.